tezos/encoding-derive/src/nom.rs

```diff
@@ -115,19 +115,14 @@ fn generate_bytes_nom_read(span: Span) -> TokenStream {
 
 fn generate_struct_nom_read(encoding: &StructEncoding) -> TokenStream {
     let generate_nom_read = match encoding.fields.len() {
-        0 => generate_struct_no_fields_nom_read,
+        0 => unreachable!("No decoding for empty struct"),
```

Comment:
@akoptelov could this Be triggered in runtime or just in compile tíme?

Reply:
Compile time only.